src/Auth/AuthUtils.php

```diff
@@ -23,7 +23,7 @@ public static function parseAuthToken(string $authToken): object
         }
 
         try {
-            list($header, $payload, $signature) = $exploded;
+            $payload = $exploded[1];
```

Comment:
What does this do?

Reply:
I only use the `$payload`, so this gets rid of a warning about the other two variables being unused.